code/modules/admin/verbs/resize.dm

```diff
@@ -0,0 +1,13 @@
+/client/proc/resize(var/mob/living/L in player_list)
```

Comment:
And will certainly change this, I was just using Smite verb as an example while doing this, since it was difficult to find proper verb examples in admin stuff.